examples/mp-guarantees/src/main.rs

```diff
@@ -365,6 +368,175 @@ fn test_overhead(config: &TestConfig, guarantee: &str, faulty: bool) -> TestResu
     Ok(true)
 }
 
+fn mc_goal_got_two_messages() -> GoalFn {
+    boxed!(|state| {
```

Comment:
Без указания типа моя IDE не видит методы `state`, поэтому предлагаю прописать

```suggestion
    boxed!(|state: &McState| {
```

Reply:
Все еще актуально

Reply:
ага, что-то проморгал при переносе, поправил